### DISCUSSION TITLE
Babel is still polyfilling object spread despite babel target set to Chrome >= 90 (Worker issue?)

### DISCUSSION THREAD

**Discussion by mfbx9da4** [vercel/next.js#27895] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

14

### What browser are you using?

chrome

### What operating system are you using?

macOs

### How are you deploying your application?

Vercel

### Describe the Bug

I want to use latest javascript syntax natively. I don't want babel to polyfill `_objectSpread`. I've set .babelrc to

```
{
  "presets": [
    [
      "next/babel",
      {
        "preset-env": {
          "useBuiltIns": false,
          "targets": "Chrome >= 90"
        }
      }
    ]
  ]
}
```

If you clone this and run it locally and profile in performance tab of chrome you will see the following 

![image](https://user-images.githubusercontent.com/1690659/128630811-850a24ca-fe23-4d45-8023-b736637d8a06.png)

![image](https://user-images.githubusercontent.com/1690659/128630824-046d5d78-755a-49af-9684-3b8e4f64cfe2.png)

Why is `_objectSpread` being polyfilled?

Is it because this code is in the worker?

https://github.com/mfbx9da4/countdown-solver/tree/main
https://countdown-solver-97i0y4mi1-mfbx9da4.vercel.app/

### Expected Behavior

For _objectSpread not to be polyfilled and to have the `{...` syntax.

### To Reproduce

Detailed in description.

**Comment by LukasBombach** [comment]
I believe Next ships a pre-compiled babel-runtime 

https://github.com/vercel/next.js/blob/2b615103af28df1324d6f06a4e82a66cb0a56728/packages/next/src/server/config-utils.ts#L134

**Comment by LukasBombach** [comment]
On the server, the babel-runtime polyfiles make up for >=9% of the rendering time
